cloud/kubernetes/README.md

```diff
@@ -186,11 +221,16 @@ If you created a secure cockroach cluster, run:
 kubectl create -f example_app_secure.yaml
 ```
 
-For every pod being created, you will need to approve its client certificate request:
+When the first pod is being initialized, you will need to approve its client certificate request:
 ```shell
-kubectl certificate approve client.root-example-secure-etc..
+kubectl certificate approve client.root
 ```
 
+If more pods are then added through `kubectl scale deployment example-secure --replicas=X`, the generated
+certificate will be reused.
+**WARNING**: the example app in secure mode should be started with only one replica, or concurrent and
```

Comment:
Yeah, I'd probably go with something like that. Although to really be safe I think you'll need a bit more logic. It's possible that the CSR could have been rejected, or that it was approved but the client that created the CSR was taken down before it could create the secret with it so the secret will never be created. You could probably do something like:

1. Try to create CSR. If it creates successfully, wait for CSR to be approved.
1. If it already exists, get it. If it's still pending, wait for it to be approved.
1. If the previous CSR was rejected, create a new CSR (by deleting the old one I guess?) and wait for it to be approved.
1. If the previous CSR was already approved, go ahead and create the secret for it.